### ISSUE TITLE
False positive for Missing $domain arg

### ISSUE THREAD

**Issue by Chouby** [WordPress-Coding-Standards/WordPress-Coding-Standards#1266]
I activated the text domain verification with:
```
<rule ref="WordPress.WP.I18n">
	<properties>
		<property name="text_domain" type="array" value="polylang" />
	</properties>
</rule>
```
and excluded the `WordPress.WP.I18n.LowLevelTranslationFunction` sniff.

However the code
`$mo->translate( $string )`
generates an error: `Missing $domain arg. (WordPress.WP.I18n.MissingArgDomain)`.

I would expect this error for a missing domain in a call to `__()` or any high level I18n functions, but the `translate` method of the `Translations` WordPress class doesn't have any domain argument. See: https://github.com/WordPress/WordPress/blob/4.9.1/wp-includes/pomo/translations.php#L94-L98, so I don't expect that using a `translate` method with only one argument would generate such error.

I am using PHPCS 2.9.1 and WPCS 0.14.

**Comment by jrfnl** [comment]
@Chouby Thanks for reporting this. Looks like the sniff is confusing your method call, with a function call to a function of the same name which also exists in WP (and does expect the `$domain` argument).
See: https://developer.wordpress.org/reference/functions/translate/

**Comment by Chouby** [comment]
I did not remember that there was also a function `translate()` in WordPress. Thanks for the fix!
